ocamltest/ocaml_tests.ml

```diff
@@ -22,10 +22,12 @@ open Ocaml_actions
 let bytecode =
   let opt_actions =
   [
+(*
     setup_ocamlc_opt_build_env;
     ocamlc_opt;
     check_ocamlc_opt_output;
     compare_bytecode_programs
+*)
```

Comment:
This diff on ocaml_tests.ml ideally shouldn't be commited, ^^ (though I agree that the current setup for testing only ocamlc.byte/ocamlopt.byte is annoying).